infra/index.ts

```diff
@@ -0,0 +1,37 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as aws from "@pulumi/aws";
+import * as awsx from "@pulumi/awsx";
+
+import exampleHandler from "../src/lambda";
+
+type Stage = "dev" | "beta" | "prod";
```

Comment:
should we declare `Stage` and `createResourceName` outside of this template so we're not redeclaring it for every stack?

Reply:
Yep eventually we can have a shared NPM package, but I assume that will be some setup so let's go with this for now.